index.html

```diff
@@ -0,0 +1,5 @@
+---
+title: "Acronsminerals"
```

Comment:
Let's fix this at all the instances

```suggestion
title: "Acrons Minerals"
```